ekf_slam_node:
  ros__parameters:
    init_p: [49.008645, 8.398104, 112.990593]
    pole0: [49.00877, 8.39774, 112.586540]
    pole1: [49.00883, 8.39754, 112.513474]
    pole2: [49.00891, 8.39730, 112.354141]
    pole3: [49.00899, 8.39705, 112.197662]
    pole4: [49.00907, 8.39678, 111.924057]
    pole5: [49.00913, 8.39658, 111.895889]
    pole6: [49.00921, 8.39634, 111.589226]
    pole7: [49.00931, 8.39604, 114.621796]
    pole8: [49.00936, 8.39585, 115.114571]
    pole9: [49.00941, 8.39567, 115.218834]
    pole10: [49.00949, 8.39544, 115.322472]
    pole11: [49.00954, 8.39524, 115.407509]
